### PR TITLE
Refine help and error message

### LINKED ISSUE #320: [BUG] "vela app run" doesn't work
```
✗ vela app run bundle1
Launching App Bundle "bundle1"
Error: name is required
```

### PR DESCRIPTION
**Pull request by zzxwill**
refined error message for `vela app run`
and correct help message for `vela comp ls`

To fix #320

### CODE REVIEW

**Review by wonderflow — APPROVED** [review]

File: pkg/application/app.go [hunk 1]
```diff
@@ -134,7 +134,7 @@ func (app *Application) Validate() error {
 		return errors.New("app is nil")
 	}
 	if app.Name == "" {
-		return errors.New("name is required")
+		return errors.New("please provide an existed App name")
```
Comment (ryanzhang-oss) [inline pkg/application/app.go:137]: What does this mean?

File: pkg/application/app_test.go [hunk 2]
```diff
@@ -119,7 +119,7 @@ components:
 			raw:           yaml3,
 			ExpName:       "",
 			InValid:       true,
-			InvalidReason: errors.New("name is required"),
+			InvalidReason: errors.New("please provide an existed App name"),
```
Comment (ryanzhang-oss) [inline pkg/application/app_test.go:122]: What does this mean?

**Review by ryanzhang-oss — CHANGES_REQUESTED** [review]

File: pkg/application/app.go (hunk 1 above)
Comment (zzxwill) [reply pkg/application/app.go:137]: I intended to provide a more specific error message for `vela app run`.

File: pkg/application/app_test.go (hunk 2 above)
Comment (zzxwill) [reply pkg/application/app_test.go:122]: For details, please refer to https://github.com/oam-dev/kubevela/issues/320:)

**Review by ryanzhang-oss — APPROVED** [review]

**Pull request merged**